js/main.js

```diff
@@ -2,25 +2,83 @@
 
 var $form = document.querySelector('form');
 
-var $url = document.querySelector('#Photo-URL');
+var $url = document.querySelector('#photo-url');
 
 var $image = document.querySelector('.image');
 
 function handleInput(event) {
-  $image.src = $url.value;
+  $image.setAttribute('src', $url.value);
 }
 
 $url.addEventListener('input', handleInput);
 
 $form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var object = {};
+
+  object.title = $form.elements.title.value;
+  object.url = $form.elements.url.value;
+  object.notes = $form.elements.notes.value;
+  object.nextid = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.unshift(object);
 
-  var entries = {};
-  entries.title = $form.elements.title.value;
-  entries.url = $form.elements.url.value;
-  entries.notes = $form.elements.notes.value;
-  entries.nextid = data.nextEntryId++;
-  data.entries.unshift(entries);
+  renderList(data.entries[0]);
   $form.reset();
   $image.src = 'images/placeholder-image-square.jpg';
-  event.preventDefault();
+
+});
+var $ul = document.querySelector('ul');
+function renderList(entry) {
```

Comment:
This function should take in an entry, and return an `li` element. This function should not append anything to the DOM